src/content/blog/ddev-oct-2024-newsletter.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "DDEV October 2024 Newsletter"
+pubDate: 2024-10-01
+#modifiedDate: 2024-09-06
+summary: DDEV October 2024 Newsletter
+author: Randy Fay
+#featureImage:
+#  src: /img/blog/2024/09/ddev-blog.png
+#  alt: DDEV running in the GitLab Pipeline
+#  credit: "[Milad Fakurian](https://unsplash.com/de/@fakurian) (Background)"
+categories:
+  - Community
+---
+
+
+**Happy October from DDEV**! I just got back from DrupalCon Barcelona and things are finally starting to cool down here in Palisade, Colorado, USA.
+
+**Podcast**! [Chad Carlson](https://www.linkedin.com/in/chadwcarlson/) of [DDEV Lead Sponsor](https://ddev.com/blog/platform-sh-becomes-a-lead-sponsor-of-ddev/) [Platform.sh](http://Platform.sh) had me on the Change Mode podcast and we had loads of fun. Learn more about the history of DDEV (and my TRS-80 Model 1 Level 2) on the [Change Mode Podcast](https://www.podcastics.com/podcast/episode/chmod-106-code-community-and-ddev-randy-fays-journey-311577/).
+
+**DrupalCon Barcelona**: I got to see lots of friends old and new in Barcelona. Four training sessions, a full-day contribution mentoring session, and loads of fun. [**Read about it on ddev.com](https://ddev.com/blog/drupalcon-barcelona-2024).**
```

Comment:
This link won't work until I pull the other blog. It mostly needs a featured Image.